src/main/java/org/usfirst/frc4904/auton/AutonRoutine.java

```diff
@@ -0,0 +1,40 @@
+package org.usfirst.frc4904.auton;
+
+import java.util.ArrayList;
+
+import org.usfirst.frc4904.standard.commands.chassis.SimpleSplines;
+
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+
+public class AutonRoutine extends SequentialCommandGroup {
+    public ArrayList<Command> commandsList;
+
+    @Override
+    public SequentialCommandGroup andThen(Command... commands) {
+        SequentialCommandGroup group = super.andThen(commands);
+        for (Command command : commands) {
+            commandsList.add(command);
+        }
+        return group;
+    }
+
+    public ArrayList<Command> getCommandsList() {
+        return commandsList;
+    }
+
+    /**
+     * Used for the spline visualizer.
```

Comment:
Question: for the visualizer, don't we also want to know things like `ChassisTurn`?

Reply:
A ChassisTurn shouldn't affect the visualization, because each spline already specifies a start and end rotation.